roles/internal/activemq/tasks/install.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Clean activemq path
+  file:
+    state: absent
+    path: "{{ activemq_install_root }}/activemq"
+
+- name: Download and unarchive activemq
+  unarchive:
+    remote_src: yes
+    src: http://archive.apache.org/dist/activemq/{{ activemq_version }}/apache-activemq-{{ activemq_version }}-bin.tar.gz
+    dest: "{{ activemq_install_root }}"
+    owner: "{{ activemq_user }}"
+    group: "{{ activemq_group }}"
+
+- name: Move activemq
+  command: mv "{{ activemq_install_root }}/apache-activemq-{{ activemq_version }}" "{{ activemq_install_root }}/activemq"
```

Comment:
This needs to be made idempotent somehow, perhaps using the when or creates statement statement.

Perhaps something like this: 
```yaml
- name: Move activemq
  command: mv "{{ activemq_install_root }}/apache-activemq-{{ activemq_version }}" "{{ activemq_install_root }}/activemq"
  args:
    creates: "{{ activemq_install_root }}/activemq"
```